src/components/Capacity.svelte

```diff
@@ -20,6 +21,8 @@
   let blockNumber = 0n;
   storeBlockNumber.subscribe((val) => (blockNumber = val));
 
+  $: showStakeToProvider = false;
```

Comment:
This is just a local variable as it doesn't have any reactive inputs.

Reply:
fixed